### PR TITLE
neuro-conda-2023b

### PR DESCRIPTION
**Pull request by joschaschmiedt**
## neuro-conda-2023b

2nd release of neuro-conda with a few new packages and updates.

## Added packages

- pynapple (#37)
- spyking-circuse (#24)
- neurotic (#22)
- mat73 (#35)
- spikeinterface-gui (#21)
- suite2p (#39)
- black, pylint, flake (#34)
- Brian2 (#40)
- NeuroKit2 (#41)

## Updated packages

...

### CODE REVIEW

**Comment by joschaschmiedt** [comment]
I'd like to make the README a bit more compact. What do you think about grouping the packages like this, @pantaray:

Currently neuro-conda includes the following neuroscientific Python packages (in alphabetic order):

- **Electrophysiology (spikes + LFP)**:   [elephant](https://elephant.readthedocs.io),   [viziphant](https://viziphant.readthedocs.io),  [esi-syncopy](https://syncopy.readthedocs.io),  [neurotic](https://neurotic.readthedocs.io/en/latest/),  [pynapple](https://pynapple-org.github.io/pynapple/),  [spyking-circus](https://spyking-circus.readthedocs.io/en/latest/),  [spikeinterface](https://spikeinterface.readthedocs.io),  [spikeinterface-gui](https://github.com/SpikeInterface/spikeinterface-gui)

- **Electrophysiology (LFP + M/EEG)**:  [bycycle](https://bycycle-tools.github.io),   [fooof](https://fooof-tools.github.io),  [neurodsp](https://neurodsp-tools.github.io),  [mne](https://mne.tools) (including the extensions     [alphaCSC](https://alphacsc.github.io/stable/index.html),    [eelbrain](https://eelbrain.readthedocs.io/en/stable/index.html),    [invertmeeg](https://github.com/LukeTheHecker/invert) (only on Linux),    [mne-ari](https://github.com/john-veillette/mne-ari),    [mne-bids](https://mne.tools/mne-bids/stable/index.html),    [mne-bids-pipeline](https://mne.tools/mne-bids-pipeline/1.1/index.html),    [mne-connectivity](https://mne.tools/mne-connectivity/stable/index.html),    [mne-features](https://mne.tools/mne-features/),    [mne-icalabel](https://mne.tools/mne-icalabel/stable/index.html),    [mnelab](https://mnelab.readthedocs.io/en/latest/index.html),    [mne-microstates](https://github.com/wmvanvliet/mne_microstates),    [mne-nirs](https://mne.tools/mne-nirs/stable/index.html),    [mne-realtime](https://mne.tools/mne-realtime/),    [mne-rsa](https://users.aalto.fi/~vanvlm1/mne-rsa/),    [pactools](https://pactools.github.io/index.html),    [pyprep](https://github.com/sappelhoff/pyprep),    [sesameeg](https://pybees.github.io/sesameeg/)),

- **Neuroimaging**:   [dipy](https://dipy.org/),  [intensity-normalization](https://intensity-normalization.readthedocs.io/en/latest/readme.html)

- **File/data formats**:  [neo](https://neo.readthedocs.io),  [nibabel](https://nipy.org/nibabel/),   [napari-nibabel](https://nipy.org/packages/napari-nibabel/index.html),  [nixio](https://nixio.readthedocs.io),  [pybids](https://bids-standard.github.io/pybids/),  [pydicom](https://pydicom.github.io/) +  [deid](https://pydicom.github.io/deid/),
  [pynwb](https://pynwb.readthedocs.io)
 
- **Machine Learning**:   [nilearn](https://nilearn.github.io/stable/index.html),  [nipype](https://nipype.readthedocs.io/en/latest/),  [nitime](https://nipy.org/nitime/) (only on Windows and Linux),
  [PyTorch](https://pytorch.org),  [scikit-learn](https://scikit-learn.org),  [tensorflow](https://www.tensorflow.org),

- **Parallelization**:  [dask](https://www.dask.org),  [esi-acme](https://esi-acme.readthedocs.io)

More neuroscience tools will be added in the future. Please open an [issue](https://github.com/neuro-conda/neuro-conda/issues) or [pull request](https://github.com/neuro-conda/neuro-conda/pulls) if you'd like a specific package
to be included.

- **Other** useful packages are (see [the environment files](/envs) for details): [black](https://black.readthedocs.io/en/stable/), [flake8](https://flake8.pycqa.org/en/latest/index.html#) + [pep8-naming](https://github.com/PyCQA/pep8-naming), [HDF5](https://www.hdfgroup.org/solutions/hdf5/) + [h5py](https://www.h5py.org/), [imageio](https://imageio.readthedocs.io/en/stable/), [jupyter](https://jupyter.org), [NumPy](https://numpy.org/), [SciPy](https://scipy.org/), [Matplotlib](https://matplotlib.org/) + [Seaborn](https://seaborn.pydata.org/),
[mat 7.3](https://github.com/skjerns/mat7.3), [R](https://www.r-project.org/),

**Comment by pantaray** [comment]
> I'd like to make the README a bit more compact. What do you think about grouping the packages like this, @pantaray:

Looks fantastic! Good idea!
